packages/blade/src/components/Accordion/Accordion.tsx

```diff
@@ -48,6 +48,7 @@ const MIN_WIDTH: BoxProps['minWidth'] = {
 };
 
 const MAX_WIDTH: BoxProps['maxWidth'] = {
+  // 100% of viewport width - 40px
```

Comment:
why `-40`?

Reply:
This is from [design spec](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-619694&t=esa0UUdu7yhP7Lcs-4)

![image](https://github.com/razorpay/blade/assets/6682655/e1590682-3193-41c1-b438-1441685cba9c)